### PR TITLE
🐛 [amp story] Change toggleAttribute to set/removeAttribute

### PR DESCRIPTION
**Pull request by processprocess**
toggleAttribute is [not widely supported](https://caniuse.com/?search=toggleAttribute) so it fails on multiple platforms.

This PR uses the toggleAttribute utility function instead.

Fixes [#120](https://github.com/ampproject/error-reporting/issues/120), [#93](https://github.com/ampproject/error-reporting/issues/93)

### CODE REVIEW

File: extensions/amp-story-page-attachment/0.1/amp-story-draggable-drawer.js [hunk 1]
```diff
@@ -237,7 +237,9 @@ export class DraggableDrawer extends AMP.BaseElement {
       ? this.startListeningForTouchEvents_()
       : this.stopListeningForTouchEvents_();
 
-    this.headerEl.toggleAttribute('desktop', !isMobile);
+    isMobile
+      ? this.headerEl.removeAttribute('desktop')
+      : this.headerEl.setAttribute('desktop', '');
```
Comment (jridgewell) [inline extensions/amp-story-page-attachment/0.1/amp-story-draggable-drawer.js:242]: See `toggleAttribute` in https://github.com/ampproject/amphtml/blob/bcf39f30a3b23fcb06ac25a875206a38567a3c45/src/core/dom/index.js#L457

Comment (processprocess) [reply extensions/amp-story-page-attachment/0.1/amp-story-draggable-drawer.js]: Awesome. Thank you Justin.

File: build-system/test-configs/forbidden-terms.js [hunk 2]
```diff
@@ -810,6 +810,7 @@ const forbiddenTermsSrcInclusive = {
   '\\.pageXOffset(?!_)': bannedTermsHelpString,
   '\\.pageYOffset(?!_)': bannedTermsHelpString,
   '\\.innerWidth(?!_)': bannedTermsHelpString,
+  '\\.toggleAttribute(?!_)': 'please use `toggleAttribute()` from core/dom',
```
Comment (processprocess) [inline build-system/test-configs/forbidden-terms.js:813]: @jridgewell I feel like we should add this to forbidden terms to prevent future errors. WDYT?

Comment (jridgewell) [reply build-system/test-configs/forbidden-terms.js:813]: SGTM.

**Review by gmajoulet — APPROVED** [review]

**Review by jridgewell — APPROVED** [review]

**Review by mszylkowski — APPROVED** [review]

**Pull request merged**